baseplate/server/__init__.py

```diff
@@ -208,6 +209,9 @@ def load_app_and_run_server():
     if args.reload:
         reloader.start_reload_watcher(extra_files=[args.config_file.name])
 
+    # clean up leftovers from initialization before we get into requests
+    gc.collect()
+
```

Comment:
I'm a little confused about what we are cleaning up here if we're are saying that we no longer have cycles and the ref counter can handle things.

Reply:
This is just a one time thing during startup. There are definitely ref cycles in the libraries we use that show up in this stage. For example, the CQL driver and CQLMapper have all sorts during init. By cleaning up here, we make sure they don't bother real requests or show up in our monitoring of steady-state garbage collection.

Reply:
I see. I forgot about other libraries. Sounds good